src/bin/pg_autoctl/cli_common.c

```diff
@@ -1311,3 +1321,168 @@ cli_pg_autoctl_reload(const char *pidfile)
 
 	return true;
 }
+
+
+/*
+ * cli_node_metadata_getopts parses the command line options for the
+ * pg_autoctl set node metadata command.
+ */
+int
+cli_node_metadata_getopts(int argc, char **argv)
+{
+	KeeperConfig options = { 0 };
+	int c, option_index = 0, errors = 0;
+	int verboseCount = 0;
+
+	static struct option long_options[] = {
+		{ "pgdata", required_argument, NULL, 'D' },
+		{ "nodename", required_argument, NULL, 'n' },
+		{ "hostname", required_argument, NULL, 'h' },
+		{ "pgport", required_argument, NULL, 'p' },
+		{ "json", no_argument, NULL, 'J' },
+		{ "version", no_argument, NULL, 'V' },
+		{ "verbose", no_argument, NULL, 'v' },
+		{ "quiet", no_argument, NULL, 'q' },
+		{ "help", no_argument, &help_flag, 0 },
```

Comment:
For consistency and usability sake. I think it's nice to keep `-h` as an alias for `--help`. We can use `H` for hostname I guess.

Reply:
Well that's the thing, `'H'` is already `--pghost`, the hostname we use to connect to the local instance of Postgres (typically we use Unix sockets in either `/var/run/postgres` or `/tmp` depending on the platform). But we don't document the short options anyway, and even mentioned getting rid of them at some point maybe?

Reply:
I think it's better to keep `'h'` for help then, and use a random other letter (like `'N'`) for hostname? As long as we don't expose it through the shortflag string I think that's fine.

Reply:
Fair, doing that soon, thanks!